Gruntfile.js

```diff
@@ -117,16 +72,21 @@ module.exports = function (grunt) {
           {expand: true, cwd: 'src/', src: ['**'], dest: JS + '/'}
         ],
         options: {
-          process: function (content, srcpath) {
-            content = substituteWidgetVersion(content);
-            if (srcpath.indexOf('copyright.frag') > 0) {
-              return content;
-            } else {
-              return content.replace(COPYRIGHT_TEXT, '');
-            }
+          process: function (content) {
+            return substituteWidgetVersion(content);
           }
         }
       },
+      courage: {
+        files: [
+          {expand: true, cwd: 'node_modules/@okta/courage/src/', src: ['**'], dest: JS + '/shared/'}
+        ]
+      },
+      'courage-vendor': {
```

Comment:
Is there a reason we split this out into a different target as opposed to putting it in the courage target?